subprojects/core/src/integTest/groovy/org/gradle/api/file/FileCollectionSymlinkIntegrationTest.groovy

```diff
@@ -356,16 +352,16 @@ class FileCollectionSymlinkIntegrationTest extends AbstractIntegrationSpec {
         when:
         run 'brokenClasspathInput'
         then:
-        fails ':brokenClasspathInput'
+        executedAndNotSkipped ':brokenClasspathInput'
 
         where:
         classpathType << [Classpath, CompileClasspath]
     }
 
-    @PendingFeature
     @Issue('https://github.com/gradle/gradle/issues/9904')
-    def "fails task with a #classpathType.simpleName File input"() {
-        def brokenInputFile = file('BrokenInputFile.class').createLink("BrokenInputFile.target")
+    def "task with a broken #classpathType.simpleName directory input is accepted"() {
+        def classes = file('classes').createDir()
```

Comment:
does the folder name actually matter here?

Reply:
I does not. It still makes sense to have a nice one.